### PR TITLE
`azurerm_mssql_database` - support for the `import` property.

### PR DESCRIPTION
**Pull request by ziyeqf**
Test
---
```
TF_ACC=1 go test -v ./internal/services/mssql -run=TestAccMsSqlDatabase_bacpac -timeout=600m       
=== RUN   TestAccMsSqlDatabase_bacpac
=== PAUSE TestAccMsSqlDatabase_bacpac
=== CONT  TestAccMsSqlDatabase_bacpac
--- PASS: TestAccMsSqlDatabase_bacpac (1187.63s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/mssql 1188.778s
```

### CODE REVIEW

File: internal/services/mssql/mssql_database_resource.go [hunk 1]
```diff
@@ -775,6 +813,54 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
...
+						ValidateFunc: validation.StringInSlice([]string{
+							string(sql.StorageKeyTypeSharedAccessKey),
+							string(sql.StorageKeyTypeStorageAccessKey),
+						}, false),
+					},
+					"administrator_login": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+					},
+					"administrator_login_password": {
```
Comment (katbyte) [inline internal/services/mssql/mssql_database_resource.go:843]: this should be
```suggestion
					"administrator_password": {
```

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/mssql/mssql_database_resource.go (hunk 1 above)
Comment (ziyeqf) [reply internal/services/mssql/mssql_database_resource.go:843]: It's the password of sql server and its schema in `azurerm_mssql_server` is `administrator_login_password`, should we make them different?
https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/mssql/mssql_server_resource.go#L78

Comment (katbyte) [reply internal/services/mssql/mssql_database_resource.go:843]: ahh it should be just password as login is redundant but as it already exists in schema not important enough to change, thx

**Review by katbyte — APPROVED** [review]
LGTM, thou i do have 1 final idea: as create mode defaults to default which is what you need for importing we could just make it conflict with import so the user can't actually set it to something other than default? then we don't have to document it/check on create and get plan time validation? 🤔  WDYT?

**Comment by ziyeqf** [comment]
> LGTM, thou i do have 1 final idea: as create mode defaults to default which is what you need for importing we could just make it conflict with import so the user can't actually set it to something other than default? then we don't have to document it/check on create and get plan time validation? 🤔 WDYT?

Thank you for the idea, it does make sense. I have refactored the code :).

**Review by katbyte — APPROVED** [review]
looks good @ziyeqf LGTM ☁️

**Pull request merged**